src/audio/dai.c

```diff
@@ -473,6 +473,14 @@ static int dai_params(struct comp_dev *dev,
 			comp_err(dev, "dai_params(): failed to alloc dma buffer");
 			return -ENOMEM;
 		}
+
+		/*
```

Comment:
The comment applies to the next line, could be rephrased a bit to refer to hw-params (and req'ed dma buffer settings) vs. params.

Reply:
rephrased